hw/ip/aes/doc/_index.md

```diff
@@ -149,8 +166,13 @@ The AES unit operates as follows:
     4. AddRoundKey Transformation: The round key is XORed with the output of the MixColumns operation and stored back into the State register.
        The 128-bit round key itself is extracted from the current value in the Full Key register.
        In parallel, the full key used for the next round is computed on the fly using the key expand module.
-1. Finally, the AES unit performs the final encryption/decryption round in which the MixColumns operation is skipped.
-   The output of this final round is forwarded to the output register in the CSRs and not stored back into the State register.
+    If running in CTR mode, the counter module iteratively updates the IV in parallel to the cipher core performing encryption/decryption.
+    Internally, the counter module uses one 16-bit counter, meaning it requires 8 clock cycles to increment the 128-bit counter value stored in the IV register.
+1. Finally, the AES cipher core performs the final encryption/decryption round in which the MixColumns operation is skipped.
+   Depending on the cipher mode, the output of this final round is potentially XORed with either the value in the IV registers (CBC decryption) or the value stored in the previous input data register (CTR mode), before being forwarded to the output register in the CSRs.
+   The output is not stored back into the State register.
+   The internal State register is cleared.
```

Comment:
Cleared to random or cleared to zero?

Reply:
It is cleared to zero. I will prepare a PR to clear this to a (pseudo-)random value.